src/yunohost/app.py

```diff
@@ -1254,11 +1253,11 @@ def _fetch_app_from_git(app):
                 branch = app[tree_index+6:]
             try:
                 subprocess.check_call([
-                    'git', 'clone', '--depth=1', url, app_tmp_folder])
+                    'git', 'clone', '--depth=1', url, extracted_app_folder])
```

Comment:
I think - since it's not easy to review that with this view - `extracted_app_folder` has not been declared here...